docusaurus/docs-internals/editor-controls/multivalue-controls/animation.md

```diff
@@ -0,0 +1,211 @@
+  type: EffectType;
+  duration: number;
+  delay: number;
+  infiniteAnimation: false;
+  big?: boolean;
+  direction?: string;
+  style?: string;
+}
+
+EffectType = "none" | "bounce" | "fade" | "rotate" | "slide" | "zoom" | "attention" | "pulse" 
+  | "wobble" | "buzz" | "scale" | "skew" | "move"
+```
+
+`type` - the type of the animation; <br/>
+`duration` - the duration of the animation in milliseconds; <br/>
+`delay` - the delay before the animation starts, in milliseconds; <br/>
+`infiniteAnimation` - indicates whether the animation should loop indefinitely; <br/>
+`big?` - specific to the `"fade"` animation, specifies if the animation will start outside the viewport; <br/>
+`direction?` - returned for all animations except `"attention"`, indicates the direction from which or to which the animation will be executed; <br/>
+`style?` - specific to the `"attention"` animation, variants of the `"attention"` animation; <br/>
+
+### Usage
+
+#### Basic example
+
+Standard definition with only the required keys. This control will be displayed on all devices.
+
+```js
+{
+  id: "animation",
+  type: "animation"
+}
+```
+
+#### Label example
+
+Adds a label to the top of the list of animations.
+
+```js
+{
+  id: "animation",
+  label: "Animations",
+  type: "animation"
+}
+```
+
+#### Devices examples
+
+It will be rendered on all devices. This value can be skipped because it is set to `"all"` by default.
+
+```js
+{
+  id: "animation",
+  type: "animation", 
+  devices: "all"
+}
+```
+
+Rendering will occur only on `desktop`.
+
+```js
+{
+  id: "animation",
+  type: "animation", 
+  devices: "desktop"
+}
+```
+
+The display is limited to responsive modes, specifically `tablet` and `mobile`.
+
+```js
+{
+  id: "animation",
+  type: "animation", 
+  devices: "responsive"
+}
+```
+
+#### Disabled example
+
+Control will be disabled. Normally, here should be your dynamic condition.
+
+```js
+{
+  id: "animation",
+  type: "animation", 
+  disabled: true
+}
+```
+
+Control will be disabled when `videoType` variable will be `"custom"`.
+`getValue` is a getter function that allows us to retrieve the value of controls by their id.
+`"videoType"` is the id of the `"select"` control below.
+
+```js
+const getToolbarContols = ({ getValue }) => {
+  const videoType = getValue("videoType");
+
+  return [
+    {
+      id: "videoType",
+      type: "select",
+      choices: [
+        { title: "Youtube", value: "youtube" },
+        { title: "Custom", value: "custom" }
+      ]
+    },
+    {
+      id: "animation",
+      type: "animation",
+      disabled: videoType === "custom"
+    }
+  ]
+}
+```
+
+#### Config type example
+
+Specifies the list of animation types that will be available for use.
+
+```js
+{
+  id: "animation",
+  type: "animation", 
+  config: {
+    type: ["wobble", "buzz", "scale"]
+  }
+}
+```
+
+#### Config replay example
+
+Enables the ability to replay the animation by clicking a special button. In this case the button will render.
```

Comment:
about "In this case the button will render"

as i see from code button will not render if `replay` is false
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/c06314f4-981f-4cc3-852b-0c387089833a)

Reply:
missed the `not`